test/xcribe/writter_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Xcribe.WritterTest do
+  use ExUnit.Case, async: false
+
+  import ExUnit.CaptureIO
+
+  alias Xcribe.Writter
+
+  @output_path "/tmp/test"
+  @invalid_output_path "/root/null"
+
+  describe "write/1" do
+    test "writes given text to set output directory" do
+      Application.put_env(:xcribe, :output, @output_path)
```

Comment:
Its important add a `setup` to tests to restore original `output` config (or just remove this config) to avoid mess up other test cases.

Reply:
Done :+1: